asteroids_code/BoilerPlate/Asteroid.cpp

```diff
@@ -55,60 +89,142 @@ void Asteroid::Render()
 	glEnd();
 }
 
-void Asteroid::MoveForward()
+void Asteroid::MoveForward(const float delta_time)
 {
-	float angle_radian = math_master.DegreeToRadian(angle_degree);
-	origin += Vector2(-ASTEROID_SPEED * sin(angle_radian), ASTEROID_SPEED * cos(angle_radian));
+	//calculate speed depending on the size, smallest = fastest
+	float LC_angle_radian = math_master.DegreeToRadian(angle_degree);
```

Comment:
What's LC?

Reply:
it s a convention to differentiate local variables from others variables meaning Local